### DISCUSSION TITLE
DataStore & DataObject References

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [bpmnServer/bpmn-server#62] [Ideas]
Hi @ralphhanna,

What are the uses of DataStore & DataObject References? How to configure & use them?

Thanks!

**Comment by ralphhanna** [comment]
Right now, you can use both DataStore and DataObject for illustration purposes only.
As far as implementation, you can use scripts to retrieve and save external data, however, I am open to other ideas! This makes a good discussion

**Comment by MaheshkumarSundaram** [comment]
Yeah @ralphhanna. I was looking into this in Camunda Docs too.

https://forum.camunda.io/t/how-to-stored-data-in-dataobjects/3590/3
https://forum.camunda.io/t/documentation-for-datastorereference/4333/3

Camunda also says that these elements are for illustration and documentation purposes. Their engine doesn't support these elements for now. But couldn't find why so or what's the problem behind this!
Does the BPMN standard itself allude that these elements are only for documentation purpose?

In relation to this, Where can I find official docs regarding BPMN elements and their standards? As of now, I just use Camunda docs to understand the elements and their functionalities. But is there any place where BPMN standards are defined like an official one?
